.travis.yml

```diff
@@ -16,6 +16,10 @@ env:
 
 matrix:
   include:
+    - php: 7.2
+      env:
+        - TEST_PHP_ARGS="-q -s output.txt -x --show-diff"
+        - TESTS=tests/atlas.phpt
```

Comment:
Was this added after https://github.com/mongodb/mongo-php-driver/pull/898/files#r205480285, or did I just miss this during the initial review.

Reply:
You missed it :-)